samples/subsys/ipc/ipm_mcux/src/main_master.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2018, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <device.h>
+#include <ipm.h>
+
+struct device *ipm;
+int gcounter;
+
+void ping_ipm_callback(void *context, u32_t id, volatile void *data)
+{
+	gcounter = *(int *)data;
+	/* Show current ping-pong counter value */
+	printk("Received: %d\n", gcounter);
+	/* Increment on our side */
+	gcounter++;
+	if (gcounter < 100) {
+		/* Send back to the other core */
+		ipm_send(ipm, 1, 0, &gcounter, 4);
+	}
+}
+
+void main(void)
+{
+	int first_message = 1; /* do not start from 0,
+				* zero value can't be sent via mailbox register
+				*/
+	printk("Hello World from MASTER! %s\n", CONFIG_ARCH);
+
+	/* Get IPM device handle */
+	ipm = device_get_binding("MAILBOX_0");
```

Comment:
Use MAILBOX_0_LABEL instead of hardcoding the string

Reply:
fixed